### ISSUE TITLE
Mail order delivery service

### ISSUE THREAD

**Issue by GotLag** [StardewModders/mod-ideas#209]
Something like Pelican Fiber, but instead of just dropping purchased items in the player's inventory, it delivers it 1 or more days later

Possibilities:
* Orders placed via a "computer" item, which functions much like the furniture/flooring catalogues
* Ability to mail order is unlocked by completing Community Centre/Joja Warehouse?
* Maybe only allow purchase of items the player has already shipped/collected? Or make items the player has not shipped take longer to deliver ("Sorry, we don't have a local supplier for that yet")?
* Higher price for out-of-season seeds/produce?
* May be better to deliver items by spawning them in a special chest or "delivery box" added to the farm or community centre, instead of drowning the player in mail when they place multiple orders

Basically a way to buy items out of season and without depending on the travelling cart's RNG, at the cost of having to plan ahead.
